html/elements/style.json

```diff
@@ -265,7 +265,8 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": "1"
+                "version_added": "1",
+                "notes": "Prior to 75, Firefox accepted any media (MIME) type. Starting in 75, Firefox accepts only 'text/css', per the spec."
```

Comment:
I think this is not really correct. Prior Firefox version parsed this as a MIME type string, so stuff like `text/css; charset=utf-8` would be allowed (additionally also the empty string?). Not arbitrary MIME types.

Reply:
@evilpie How would you suggest rephrasing? "Prior to 75, Firefox parsed this as a media (MIME) type string."?

Reply:
Maybe "Prior to 75, Firefox allowed any CSS MIME type with optional parameters, this has been restricted to the literal string `text/css` per spec" ?

Reply:
OK, thanks.